api/overview/azure/ai.personalizer-readme-pre.md

```diff
@@ -23,13 +23,13 @@ is a cloud-based service that helps your applications choose the best content it
 Install the Azure Personalizer client library for .NET with [NuGet][nuget]:
 
 ```dotnetcli
-dotnet add package Azure.AI.Personalizer --version 2.0.0-beta.1
+dotnet add package Azure.AI.Personalizer --version 2.0.0-beta.2
```

Comment:
Can we put '--prerelease' instead of using a specific version. That way, we don't have to update this line after every release.